1526288596346	Sync.LogManager	DEBUG	Flushing file log
1526288596358	Sync.LogManager	DEBUG	Log cleanup threshold time: 1525424596358
1526288596494	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1526288596504	Sync.Service	DEBUG	User-Agent: Firefox/59.0.2 (Linux x86_64) FxSync/1.61.0.20180406121424.desktop
1526288596504	Sync.Service	INFO	Starting sync at 2018-05-14 16:03:16 in browser session qzKXTZzD26AL
1526288596504	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1526288596508	Sync.Status	INFO	Resetting Status.
1526288596508	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1526288596508	Sync.BrowserIDManager	DEBUG	_ensureValidToken already has one
1526288596646	Sync.LogManager	DEBUG	Done deleting files.
1526288597252	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1526288597252	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1526288597254	Sync.Service	DEBUG	User-Agent: Firefox/59.0.2 (Linux x86_64) FxSync/1.61.0.20180406121424.desktop
1526288597254	Sync.Service	INFO	Starting sync at 2018-05-14 16:03:17 in browser session qzKXTZzD26AL
1526288597254	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync". (resource://services-sync/util.js:43:5) JS Stack trace: LockException@util.js:43:5 < throwLockException@util.js:119:11 < WrappedLock@util.js:133:9 < _lockedSync@service.js:1099:12 < sync/<@service.js:1091:13 < WrappedCatch@util.js:107:22 < sync@service.js:1080:12
1526288597256	Sync.Service	INFO	Cannot start sync: already syncing?
1526288598814	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1526288598814	Sync.Service	DEBUG	User-Agent: Firefox/59.0.2 (Linux x86_64) FxSync/1.61.0.20180406121424.desktop
1526288598814	Sync.Service	INFO	Starting sync at 2018-05-14 16:03:18 in browser session qzKXTZzD26AL
1526288598816	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync". (resource://services-sync/util.js:43:5) JS Stack trace: LockException@util.js:43:5 < throwLockException@util.js:119:11 < WrappedLock@util.js:133:9 < _lockedSync@service.js:1099:12 < sync/<@service.js:1091:13 < WrappedCatch@util.js:107:22 < sync@service.js:1080:12
1526288598816	Sync.Service	INFO	Cannot start sync: already syncing?
1526288603818	Sync.Resource	WARN	GET request to https://sync-600-us-west-2.sync.services.mozilla.com/1.5/92728811/info/collections failed: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1526288603818	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1526288603822	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1526288603832	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1526288603834	Sync.ErrorHandler	DEBUG	Addons installed: 8
1526288603834	Sync.ErrorHandler	DEBUG	 - Firefox Multi-Account Containers, version 6.0.0, id @testpilot-containers
1526288603834	Sync.ErrorHandler	DEBUG	 - GNOME Shell integration, version 10.1, id chrome-gnome-shell@gnome.org
1526288603834	Sync.ErrorHandler	DEBUG	 - Web Extension for Shaarli, version 2.0.0, id shaarli@imirhil.fr
1526288603836	Sync.ErrorHandler	DEBUG	 - DuckDuckGo Privacy Essentials, version 2018.5.1, id jid1-ZAdIEUB7XOzOJw@jetpack
1526288603836	Sync.ErrorHandler	DEBUG	 - uBlock Origin, version 1.16.4, id uBlock0@raymondhill.net
1526288603836	Sync.ErrorHandler	DEBUG	 - FoxyProxy Standard, version 6.2, id foxyproxy@eric.h.jung
1526288603836	Sync.ErrorHandler	DEBUG	 - Zotero Connector, version 5.0.39, id zotero@chnm.gmu.edu
1526288603836	Sync.ErrorHandler	DEBUG	 - Sync Tab Groups, version 0.6.9, id sync-tab-groups@eric.masseran
